src/common/res/features/goal-indicator/main.js

```diff
@@ -0,0 +1,106 @@
+(function poll() { 
+  // Waits until an external function gives us the all clear that we can run (at /shared/main.js)
+  if ( typeof ynabToolKit !== "undefined"  && ynabToolKit.pageReady === true ) {
+  
+    ynabToolKit.featureOptions.goalIndicator = true;
+    ynabToolKit.goalIndicator = function ()  { // Keep feature functions contained within this
+    	var entityManager = ynab.YNABSharedLib.getBudgetViewModel_AllAccountTransactionsViewModel()._result.getEntityManager();
+
+    	function addIndicator (element, inticator) {
+    		var budgetedCell = $(element).find(".budget-table-cell-budgeted");
+    		if (budgetedCell.has(".goal-indicator").length == 0) {
+    			budgetedCell.prepend('<div class="goal-indicator">' + inticator + '</div>')
+    		}
+    	}
+
+    	function pad(num, size) {
+		    var s = "0" + num;
+		    return s.substr(s.length-size);
+		}
+
+    	function getCurrentYM () {
```

Comment:
I would recommend the approach in Trello:
https://trello.com/c/ghNbTjKn/15-how-to-get-opened-budget-month-as-yyyy-mm

YNAB is technically localized, so in German "Mai" is "Mai", etc. There's actually a DateFormatter class in ynab.utilities that does localized date formatting (and will respect user preferences).